core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -497,6 +496,16 @@ class MultiCloudWorkspaceService(override val ctx: RawlsRequestContext,
       throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.NotImplemented, "MC workspaces are not enabled"))
     }
 
+    val previewDate = new DateTime(2023, 9, 13, 0, 0, DateTimeZone.UTC)
+    val isTestProfile = profile.getCreatedDate() == null || profile.getCreatedDate() == ""
```

Comment:
I didn't want to have to update all of our mocked billing profiles to include a createdDate.